websites/D/Daily Mail/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Daily Mail",
+	"description": {
+		"en": "MailOnline - get the latest breaking news, celebrity photos, viral videos, science & tech news, and top stories from MailOnline and the Daily Mail newspaper."
+	},
+	"url": "www.dailymail.co.uk",
+	"matches": [
+		"*://dailymail.co.uk/*"
```

Comment:
```suggestion
		"*://www.dailymail.co.uk/*"
```

Reply:
I didn't put this in because of the `games` subdomain, should it be `"*://*.www.dailymail.co.uk/*"`?

Reply:
In that case, yeah